examples/db-connect.php

```diff
@@ -1,37 +1,46 @@
 <?php
+declare( strict_types = 1 );
 
-$dotenv = Dotenv\Dotenv::createImmutable(dirname(__DIR__));
+use Dotenv\Dotenv;
+
+$dotenv = Dotenv\Dotenv::createImmutable( dirname( __DIR__ ) );
 $dotenv->load();
 
 use Illuminate\Database\Capsule\Manager as Capsule;
 
 $capsule = new Capsule;
-$capsule->addConnection([
-    'driver'    => 'mysql',
-    'host'      => 'localhost',
-    'database'  => getenv('DATABASE'),
-    'username'  => getenv('USERNAME'),
-    'password'  => getenv('PASSWORD'),
-    'charset'   => 'utf8',
-    'collation' => 'utf8_unicode_ci',
-    'prefix'    => '',
-]);
+$capsule->addConnection(
+	[
+		'driver' => 'mysql',
+		'host' => 'localhost',
+		'database' => getenv( 'DATABASE' ),
+		'username' => getenv( 'USERNAME' ),
+		'password' => getenv( 'PASSWORD' ),
+		'charset' => 'utf8',
+		'collation' => 'utf8_unicode_ci',
+		'prefix' => '',
+	]
+);
 
 // Make this Capsule instance available globally via static methods... (optional)
 $capsule->setAsGlobal();
 
-function savePush(APNSPush $push, $token) {
+// phpcs:ignore NeutronStandard.Globals.DisallowGlobalFunctions.GlobalFunctions,WordPress.NamingConventions.ValidFunctionName.FunctionNameInvalid
```

Comment:
Decided to ignore instead of fixing because this is just an example.

Reply:
We could ignore this – but if we do we should do it in the `.phpcs.xml` configuration once for the whole file – I have an example of that in #30.

Personally, I don't mind fixing these down the road to make everything nice, but maybe once we're up and running with better test coverage? 😆